packages/faust-nx/src/getWordPressProps.tsx

```diff
@@ -76,16 +80,19 @@ export async function getWordPressProps(options: getWordPressPropsConfig) {
     };
   }
 
-  if (template.query) {
+  if (template.query && !isPreview) {
     await client.query({
       query: template.query,
-      variables: template?.variables ? template.variables(seedNode) : undefined,
+      variables: template?.variables
+        ? template.variables(seedNode, false) // 2nd arg will always be false.
```

Comment:
I wonder if the second variable could be an optional `context` object with the `isPreview: false` instead of the `isPreview` directly. What do you think?

Reply:
Love this idea